mms/tests/unit_tests/test_model_loader.py

```diff
@@ -0,0 +1,155 @@
+# Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#     http://www.apache.org/licenses/LICENSE-2.0
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+
+import pytest
+import os
+import shutil
+import json
+from mms.model_loader import MANIFEST_FILENAME
+from mms.model_loader import ModelLoader
+
+model_dir_path = 'my-model'
+handler_file = 'handler'
+
+manifest_invalid_data_missing_paramsFile = {"model": {}, "engine": {"engineName": "MxNet"}}
+manifest_invalid_data_missing_symbolFile = {"model": {"parametersFile": "my-model/params1"},
+                                            "engine": {"engineName": "MxNet"}}
+manifest_valid_data = {"model": {"parametersFile": "my-model/params1", "symbolFile": 'my-model/symbol.json'},
+                       "engine": {"engineName": "MxNet"}}
+
+
+def empty_file(filepath):
+
+    """
+    Utility function for creating an empty file, which will be used throughout these unit tests
+    :param filepath:
+    :return:
+    """
+
+    open(filepath, 'a').close()
+
+
+@pytest.fixture(scope='session')
+def create_empty_manifest_file():
+    """
+    By setting the scope of create_empty_manifest_file as session we are ensuring that this piece of code runs once for all of the tests combined in this file
+    """
+    # Setup
+    path = '{}/'.format(model_dir_path)
+    if os.path.exists(path):
+        shutil.rmtree(path)
+    os.mkdir(path)
+    path = '{}/'.format(path) + MANIFEST_FILENAME
```

Comment:
You could use '{}/{}' or os.path.join.  There are also other places in the file where os.path.join should be used.

Reply:
Will do it.